pop-api/src/v0/fungibles/traits.rs

```diff
@@ -101,10 +106,13 @@ pub trait Psp22Metadata {
 pub trait Psp22Mintable {
 	/// Creates `value` amount of tokens and assigns them to `account`, increasing the total supply.
 	///
+	/// The selector for this message is `0x7a9da510` (first 4 bytes of
```

Comment:
Manually verified to be correct.

Reply:
<img width="1141" alt="Screenshot 2024-10-03 at 00 57 27" src="https://github.com/user-attachments/assets/dad5d1db-6b23-4da3-a39a-8f5471ad4a0f">

Confirmed with https://www.shawntabrizi.com/substrate-js-utilities/